### DISCUSSION TITLE
[IntelliSense] VSCode extension requires Config file to work

### DISCUSSION THREAD

**Discussion by cybercoder-naj** [tailwindlabs/tailwindcss#13118] [Ideas]
With TailwindCSS v4.0.0 now in alpha, it does not require any configuration file such as `tailwind.config.ts`. However, the VSCode extension requires it to provide IntelliSense.

**Comment by thedv91** [comment]
You can switch tailwindcss-intellisense to Pre-Release version

**Reply by lukemorales** [reply]
The extension still requires a config file to provide intellisense, at least in a NextJS project in my case, not sure how it behaves on a vite project

**Reply by cybercoder-naj** [reply]
I am running Nuxt3 on Vite and it is the same problem.

**Reply by lukemorales** [reply]
In my case, I was able to figure out what was happening: it's an issue in monorepo setups. If I setup a simple nextjs project, the extension works normally. I opened a [discussion](https://github.com/tailwindlabs/tailwindcss/discussions/13314) to find a solution to make it work with V4 and monorepos
